src/claim/Claimer.ts

```diff
@@ -70,7 +70,7 @@ export default class Claimer implements IClaimer {
     if (!validate(mnemonic)) {
       throw new Error('Invalid mnemonic')
     }
-    const seed = toSeed(mnemonic, password)
```

Comment:
`toSeed` was removed and "replaced" with mnemonicToMiniSecret [in 2.12.1](https://github.com/polkadot-js/common/releases/tag/v2.12.1)